networkx/algorithms/__init__.py

```diff
@@ -11,6 +11,7 @@
 from networkx.algorithms.cycles import *
 from networkx.algorithms.cuts import *
 from networkx.algorithms.dag import *
+from networkx.algorithms.disjoint_paths import *
```

Comment:
The problem with your importing is this line. You try to import a file in folder `.\networkx\tree\master\networkx\algorithms`, but you want to import `__init.py__` in folder `.\networkx\tree\master\networkx\algorithms`, which is what the line below does.